src/app/CASESessionManager.h

```diff
@@ -83,7 +84,8 @@ class CASESessionManager : public OperationalSessionReleaseDelegate, public Sess
                                 ,
                                 uint8_t attemptCount = 1, Callback::Callback<OnDeviceConnectionRetry> * onRetry = nullptr
 #endif // CHIP_DEVICE_CONFIG_ENABLE_AUTOMATIC_CASE_RETRIES
-    );
+                                ,
+                                TransportPayloadCapability transportPayloadCapability = TransportPayloadCapability::kDefault);
```

Comment:
What is "default". It's  a very relative name. Is there a better name?

Reply:
Default was meant to indicate the MRP path which is, essentially, the default path with a fixed MTU defined payload size limit. I could change it to MRPPayload if that helps clarify it better.

Reply:
Changed the kDefault to kMRPPayload.